ibis/udf/vectorized.py

```diff
@@ -20,7 +20,7 @@
 )
 
 
-def _coerce_to_tuple(
+def _coerce_to_dict(
```

Comment:
Can you remind me what this change is for?

Reply:
The comment below:

```
# Case 2: Struct output, reduction UDF -> coerce to tuple
```

Though the comment is now partially incorrect (should be coerce to `dict`), the first part indicates that the output is a struct. `dict` is the correct return type for a scalar struct value.

Previously the names of the fields in the tuple were handled elsewhere (roughly speaking, wherever `Destruct*` pseudo-expression types needed to provide names, essentially in projections or things that desugar to projections).

Reply:
kk